dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/services/ApiKeyManager.kt

```diff
@@ -10,6 +10,7 @@ import org.dataland.datalandbackendutils.apikey.ParsedApiKey
 import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
```

Comment:
There is an empty file "dataland-api-key-manager/apiKeyManagerOpenApi.json.formatted.json" in this PR.
What is it for?